src/content/1.7/project/maintainers-guide/summary-github-lifecycle.md

```diff
@@ -12,92 +12,187 @@ This page quickly describes how [Issues](https://guides.github.com/features/issu
 
-This is why so much Issues cannot be reproducted on our side, but to find it out multiple explorations and attempts are necessary.
+### Other
 
-### Support request
+Some issues do not fit in the previous categories, such as [Releases Issues](https://github.com/PrestaShop/PrestaShop/labels/Release) which are used to follow the progress of a release, or [Epic Issues](https://github.com/PrestaShop/PrestaShop/labels/Epic) that group together all requests belonging to the same subject.
 
-We sometimes receive support requests on GitHub, ranging from questions about the software to "please help me to do X in my shop" requests. GitHub is for the software development so we usually redirect users to other channels using the [Support template](https://github.com/PrestaShop/PrestaShop/issues/new?template=3_support_question.md).
+## Projects
 
-### Other
+Available in the "[Projects](https://github.com/PrestaShop/PrestaShop/projects)" tab of PrestaShop’s GitHub repository, versions kanban boards define the scope of a minor version in terms of what the Core Maintainers and Core Product teams agree must be included before that release is out.
+Both the product and the maintainers teams use it to prepare and plan the next developments and, since we deal with an open source project, the purpose of having those kanbans public is to help the community and the external mainteners to know what is at stake.
+
+### Patches versions
+
+There should only have regressions issues in the kanban of a patch version. For example, 1.7.7.1 and 1.7.7.2 (and so on...) kanbans only includes the regressions of the 1.7.7.0 version.
+
+### Minor versions
+
+There should only have "must-have" issues in the kanban of a minor/major version and it always have an owner, be it a product manager or a developer.
+Only "must-have" issues are included in the kanban, as it must reflect the progress of the version. For example, an issue reports a must-have bug prioritized for the 1.7.8.0, the issue must appear in the 1.7.8 kanban while the linked pull request must have the 1.7.8.0 milestone.
+
+### Nice to have
+
+"Nice to have" issues are part of another [kanban](https://github.com/PrestaShop/PrestaShop/projects/13) and are not intended for a particular version.
+
+## Definition of states
+
+Community contributions that fix a feature request or an improvement, must be approved by the Core product team if they bring a change in the functional behavior and by the Core developer team if they bring a change in the technical behavior. Ideally before being reviewed, at least before being merged. Here is why:
+
+It is a bug? The specifications must be completed only when it's incomplete or when there's a behavior change.
+It is a feature request or an improvement? It must be validated, prioritized, and specified. Specification can be written by a contributor or by the Core Product team. The Core Product team has the final word in whether the specification is valid or not.
```

Comment:
```suggestion
It is a feature request or an improvement? It must be validated, prioritized, and specified. Specification can be written by a contributor or by the Core Product team. The Core Product team decides whether the specification is valid or not.
```
A little suggestion to make the sentence a bit more friendly.

Your version sounds a bit like "I am the boss!" 😅